test/90_1_prm/data/prototype_list.json

```diff
@@ -10,7 +10,7 @@
         ["SmallOffice", "90.1-2013", "ASHRAE 169-2013-2A", []]
     ],
     "isresidential": [
-        ["SmallOffice", "90.1-2013", "ASHRAE 169-2013-2A", [["set_userdata_path",["userdata_ofs_01"]]]]
+        ["SmallOffice", "90.1-2013", "ASHRAE 169-2013-2A", []]
```

Comment:
This is an unintended change, should this be reverted? I didn't make it, it got pick up during one of the branch merge.

Reply:
I'd like to keep that in for now, since it is the only example of how to set up the user data for testing.

Reply:
@dmaddoxwhite - just reverted this change.